Shared/Samples/Apply class breaks renderer to sublayer/ApplyClassBreaksRendererToSublayerView.swift

```
@@ -0,0 +1,123 @@
+    
+    /// A Boolean value indicating that the class breaks renderer is applied.
+    @State private var applyClassBreaksRenderer = false
+    
+    /// The map with a topographic basemap style.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISTopographic)
+        
+        // Sets the initial viewpoint.
+        map.initialViewpoint = Viewpoint(
+            boundingGeometry: Envelope(
+                xMin: -13934661.666904,
+                yMin: 331181.323482,
+                xMax: -7355704.998713,
+                yMax: 9118038.075882,
+                spatialReference: .webMercator
+            )
+        )
+        return map
+    }()
+
+    /// The error shown in the error alert.
+    @State private var error: Error?
+
+    var body: some View {
+        MapView(map: map)
+            .task {
+                // Adds the map image layer to the map.
+                map.addOperationalLayer(mapImageLayer)
+                await loadMapImageSublayer()
+            }
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    Toggle("Change Sublayer Renderer", isOn: $applyClassBreaksRenderer)
+                }
+            }
+            .onChange(of: applyClassBreaksRenderer) {
+                countiesLayer?.renderer = if applyClassBreaksRenderer {
+                    // Applies the class breaks renderer.
+                    .populationRenderer
+                } else {
+                    // Applies the original renderer.
+                    originalRenderer
+                }
+            }
+            .errorAlert(presentingError: $error)
+    }
+    
+    func loadMapImageSublayer() async {
+        do {
+            // Loads the map image layer.
+            try await mapImageLayer.load()
+            // Gets the sublayers.
+            let mapImageSublayers = mapImageLayer.mapImageSublayers
+            guard mapImageSublayers.count >= 3 else { return }
```

This is a silent fail since the countiesLayer and originalRenderer are not set, and neither is the error. I think the two choices here are:
1. throw an error that you define
2. let it crash when you subscript the sublayer on line 84 if the wrong data is used.

---

I went with 2.